src/components/AddComment.jsx

```diff
@@ -0,0 +1,42 @@
+import { useEffect } from "react";
+import { useState } from "react";
+import "../App.css";
+import { postComment } from "../api";
+
+function AddComment({ setComments, review_id }) {
+  const [newComment, setNewComment] = useState("");
+
+  const handleChange = (event) => {
+    setNewComment(event.target.value);
+  };
+
+  const handleSubmit = (event) => {
+    event.preventDefault();
+    postComment(review_id, newComment)
+      .then((commentsApi) => {
+        setNewComment("");
+        setComments((currComments) => {
+          const newComments = [...currComments];
+          newComments.unshift(commentsApi);
+          return newComments;
+        });
+      })
+      .catch((err) => {
+        console.log(err);
+      });
+  };
+
+  return (
+    <form onSubmit={handleSubmit}>
+      <label>Add a Comment</label>
+      <textarea
+        id="newComment"
+        value={newComment}
+        onChange={handleChange}
+      ></textarea>
+      <button>Add</button>
```

Comment:
We will also want to make sure we are not able to click the post button more than once to prevent multiple posts being sent.